apps/rush-lib/src/start.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
+// See LICENSE in the project root for license information.
+
+import { EOL } from 'os';
+import * as colors from 'colors';
+
+import rushVersion from './rushVersion';
+import RushCommandLineParser from './cli/actions/RushCommandLineParser';
+
+// tslint:disable-next-line:export-name
```

Comment:
>// [](start = 0, length = 2)

Add a detailed JSDoc comment explaining that this is the main entry point for rush invoking rush-lib, and document what the parameters are and why we need them.